{"name": "test_03", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_03():\n        print('------------')\n        print('test03')\n>       assert 1 == 2\nE       assert 1 == 2\n\ntest_case03.py:19: AssertionError"}, "start": 1611063001349, "stop": 1611063001350, "uuid": "cd74c31c-4e89-4d41-9590-8b87aa90da22", "historyId": "aaeffdda95d4d34ff4d03238f03c8139", "testCaseId": "e69660cb497d57b4d0abb880f6116dec", "fullName": "test_case03#test_03", "labels": [{"name": "suite", "value": "test_case03"}, {"name": "host", "value": "DESKTOP-LTNMVII"}, {"name": "thread", "value": "16648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case03"}]}